data/apps.yaml

```diff
@@ -10,11 +10,11 @@ content:
     description: >
       A twitter bot that adds, pins, unpins your tweets to public IPFS network using IPFS Cluster.
   - title: Cyber
-    website: https://cybercongress.ai/
+    website: https://cyber.page
     source: https://github.com/cybercongress
     picture: /images/cybercongress.png
     description: >
-      A consensus computer on top of IPFS that uses IPFS hashes to create cyberlinks.
+      A decentralized google alternative, built with the help of IPFS.
```

Comment:
```suggestion
      A decentralized Google alternative, built with the help of IPFS.
```

Reply:
TBH, we ise google without the capital G on purpose. But I see what you mean. Changed google to Google and updated the readme too. Thanks